### PR TITLE
How it works page image

### PR DESCRIPTION
**Pull request by jakey-ads**
Pull request for issue #738
Hopefully I've followed the process correctly. Do let me know.
(Big shout to @pavlenex for the tutorial content 💪 )

### CODE REVIEW

File: guide/how-it-works/introduction.md [hunk 1]
```diff
@@ -8,11 +8,16 @@ permalink: /guide/how-it-works/
 main_classes: -no-top-padding
 ---
```
Comment (GBKS) [inline guide/how-it-works/introduction.md:9]:
```suggestion
image: https://bitcoin.design/assets/images/guide/how-it-works/how-it-works-preview.png
---
```
Need to reference the preview image here so it gets included in the HTML head. This is what social media sites use for link previews.

File: guide/how-it-works/introduction.md [hunk 2]
```diff
@@ -8,11 +8,16 @@ permalink: /guide/how-it-works/
...
-
-Illustration sources:
-
--->
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/how-it-works.png"
+   retina = "/assets/images/guide/how-it-works/how-it-works-@2x.png"
+   mobile = "/assets/images/guide/how-it-works/how-it-works-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/how-it-works-mobile@2x.png"
+   alt-text = "How it works header image."
```
Comment (GBKS) [inline guide/how-it-works/introduction.md:16]:
```suggestion
   alt-text = "Multiple interlocking gears, center gear has a bitcoin symbol on it"
```
Alt text should describe what is in the image for people (and computers) who can't see it.

**Comment by GBKS** [comment]
Looks great, @jakey-ads. Just had to comments you can easily accept to resolve.

**Comment by jakey-ads** [comment]
Thanks @GBKS for those tweaks. Understood 👌

**Comment by GBKS** [comment]
Sorry, I used the wording. By "resolve", I meant "commit suggestion", I was not thinking in GitHub UI language. I prepared my comments as suggestions and if you accept them, they get added to your PR.

**Comment by jakey-ads** [comment]
@GBKS Gotcha. I hit 'resolve' on both of them, as it was the only affirmative option I saw when viewing your comments. Assuming that would update the PR with your changes.

Is that ok or do I need to do something else? I'm viewing on my phone this morning so maybe I can't easily see certain options.

**Review by GBKS — APPROVED** [review]
Thanks for those final few tweaks, and sorry for being slow on this PR. Let's get this merged in and live. Big thanks for your contribution.

**Pull request merged**